charts/quickwit/values.yaml

```diff
@@ -282,6 +292,10 @@ janitor:
 
   podAnnotations: {}
 
+  serviceAnnotations: {}
+
+  serviceType: {}
```

Comment:
Same comment, we can have something like:

```yaml
janitor:
   service:
     # type: ClusterIP
     annotations: {}
```

Instead of `serviceAnnotations` and `serviceType`